### DISCUSSION TITLE
Polling mode off behaviour

### DISCUSSION THREAD

**Discussion by alexeiw123** [alandtse/tesla#241] [Q&A] [answered]
Hi All,
As I'm already using teslamate with the streaming API, I'm using it via MQTT to populate all sensor information and have disabled the sensors in this tesla integration which are duplicates for me. What I'm using this sensor for is to control locks and HVAC through HA.

Without sensor data, I have little use for polling to be on, however is switching it off going to impact those controls? If the car is in sleep and I attempt to unlock a door, does polling disabled prevent this from working?

**Comment by alandtse** [comment, marked as answer]
Without polling, commands that require knowing state won't be correct. For example locks only allow lock or unlock based on known state. If you don't use the UI and just commands, then it won't matter.

**Reply by alexeiw123** [reply]
Excellent, thank you. I am using custom lovelace that calls services so this should be OK for my use case!

**Comment by w0lf3h** [comment]
@alexeiw123 have you run into any problems doing this?
looking into a similar use case, both for climate and for interrupting charging power. I've noticed that the polling switch will turn on on every restart of HA, so I will probably end up adding an automation to turn it off on HA reboot.

**Reply by alexeiw123** [reply]
I ended up leaving polling on just with a very long polling time (3600s) have run in to no issues and car is sleeping fine. I disable the sensors that I don't need.

**Reply by w0lf3h** [reply]
I guess that would work just as well! Thanks. :)
